hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modalSize {
+    position: absolute;
+    //margin: auto;
+    margin-top: 20px;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 400px;
+    background: #fafafa;
+    boxshadow: 24;
+    p: 4;
+}
+
+.title {
+    margin-top: 30px;
+    text-align: center;
+}
+
+.description {
+    margin-top: 10px;
+    text-align: center;
+}
+
+.buttons {
+    float: right;
```

Comment:
using float is not recommended because its not predictable for responsive design. Since you're using a lot of margins in your css maybe consider using a Grid to format everything in the modal. The Grid will also take away the usage of float.

Reply:
Isn't grid for stuff like laying out arrays of cards?

Reply:
yes it can be but Grid's main purpose is formatting responsive layouts so it can be used anywhere you have a specific layout for your text/components 😀